docs/UserGuide.md

```diff
@@ -236,7 +213,31 @@ Example of usage:
 note list
 ```
 
-### 3.12. Displaying statistics: `stats`
+### 3.12. Bookmarking a question: `bookmark`
+
+Stores any question encountered in a quiz in a list.
+
+Note: 
+- Bookmarking a question can only be done during a quiz
+
+Example of usage:
+```
+bookmark
+```
+
+### 3.13. Listing out all bookmarked questions: `bookmark`
+
+Lists out all the bookmarked questions and each of their four options.
+
+Note:
+- Listing out all bookmarked questions can only be done in the main menu
+
+Example of usage:
+```
+bookmark
+```
+
```

Comment:
would be good to add some screenshots here

Reply:
Excellent idea, I shall get on it right away. Hold off on merging this branch.